pkg/tfvars/kubevirt/kubevirt.go

```diff
@@ -16,23 +15,25 @@ type config struct {
 	Storage                    string            `json:"kubevirt_master_storage"`
 	StorageClass               string            `json:"kubevirt_storage_class"`
 	NetworkName                string            `json:"kubevirt_network_name"`
+	InterfaceBindingMethod     string            `json:"kubevirt_interface_binding_method"`
 	PersistentVolumeAccessMode string            `json:"kubevirt_pv_access_mode"`
 	ResourcesLabels            map[string]string `json:"kubevirt_labels"`
 }
 
 // TFVarsSources contains the parameters to be converted into Terraform variables
 type TFVarsSources struct {
-	MasterSpecs     []*v1.KubevirtMachineProviderSpec
-	ImageURL        string
-	Namespace       string
-	ResourcesLabels map[string]string
+	MasterSpecs            []*v1.KubevirtMachineProviderSpec
+	ImageURL               string
+	Namespace              string
+	InterfaceBindingMethod string
```

Comment:
Can we pull the value from `MasterSpecs.InterfaceBindingMethod` instead of adding a new field?

Reply:
fixed, removed the new field from `TFVarsSources`